devices/Chs6540/Chs6540.cs

```diff
@@ -0,0 +1,109 @@
+// Licensed to the .NET Foundation under one or more agreements.
+// The .NET Foundation licenses this file to you under the MIT license.
+
+using System;
+using System.Device.I2c;
+using System.IO;
+
+namespace Iot.Device.Chs6540
+{
+    /// <summary>
+    /// Chs6540 touch screen
```

Comment:
Missing '.' closing the sentence (and on a few others ahead).

Reply:
we'll have **a lot** of them to correct in all the repo :-) This binding is coming from FT6xxx6x which is a clone of another one (don't remember. I'll adjust but keep in mind in most of the bindings, this has no been reinforced.